jsx/Tabs.js

```diff
@@ -171,7 +171,56 @@ TabPane.propTypes = {
   activeTab: React.PropTypes.string
 };
 
+class Modal extends React.Component {
+
+  render() {
+    // Render nothing if the "show" prop is false
+    if(!this.props.show) {
+      return null;
+    }   
+
+    //White Modal Window
+    const modalStyle = { 
+      position: 'relative',
+      maxWidth: 1000,        
+      maxHeight: '100%',
+      margin: '0 auto',
+      backgroundColor: '#fff',
+      padding: 30,
+      borderRadius: 10,
+      overflowY: 'auto',
+      zIndex: 9999,
+    };  
+
+    //Grey Background
```

Comment:
this doesn't have a grey background, it has a black background with an alpha channel